node/node.go

```diff
@@ -647,10 +650,41 @@ func (n *Node) sendFromTxAccumulator() {
 			txs = append(txs, tx)
 		}
 
+		atomic.AddUint32(&n.txSentCounter, uint32(len(txs)))
+
 		n.sendBulkTransactions(txs)
 	}
 }
 
+func (n *Node) printTxSentCounter(numSeconds int, waitTime time.Duration) {
```

Comment:
Could you please document this function? I know it is not exported but it's really hard to figure out it's purpose

Reply:
added description + TODO, the thing is not testable (yet)